platform/core/lib/src/bootstrap.rs

```diff
@@ -31,6 +32,7 @@ const RDF_DB: &str = "rdf_db";
 pub struct BoostrapArgs {
     language_code: String,
     fs_root: String,
+    update_notification_id: u32,
```

Comment:
It is an ok'ish solution since the id is stored in bundle. Otherwise that bundle file could be copied in core file system space and let core read it directly.

Reply:
Yeah, I think we might want to refactor towards that, just store the notification data in a JSON file instead of translated strings. But I feel this PR is big enough already :D